migrate.go

```diff
@@ -87,6 +87,7 @@ func (b byId) Less(i, j int) bool { return b[i].Less(b[j]) }
 type MigrationRecord struct {
 	Id        string    `db:"id"`
 	AppliedAt time.Time `db:"applied_at"`
+	DownSql   string    `db:"down_sql"`
```

Comment:
Ugh, that's a good point. I'll try to add something that detects an older database and migrate it but that feels "off", especially since package is meant to migrate databases!